<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editvisit</class>
 <widget class="QDialog" name="Editvisit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>781</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>760</width>
      <height>310</height>
     </rect>
    </property>
    <property name="title">
     <string> Visitante</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>46</y>
       <width>40</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>RG</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>87</y>
       <width>40</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CPF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>128</y>
       <width>40</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="LblPhoto">
     <property name="geometry">
      <rect>
       <x>447</x>
       <y>0</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>pessoa.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>191</width>
       <height>100</height>
      </rect>
     </property>
     <property name="title">
      <string> Entrada</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>28</y>
        <width>170</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="LblData">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="DtEdtData">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="LblHora">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Hora</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTimeEdit" name="TmEdtHora">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>164</y>
       <width>191</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string> Saida</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>26</y>
        <width>161</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="LblSaida">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Saida</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="DtEdtSaida">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Lblsaida_hora">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Hora</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTimeEdit" name="TmEdtsaida_hora">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QRadLineEditCompleter" name="lineEditRG">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>320</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCPF">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>320</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadLineEditEnter" name="lineEditVisitante">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>320</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBaterFoto">
     <property name="geometry">
      <rect>
       <x>671</x>
       <y>184</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Bater &amp;Foto</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>210</y>
       <width>311</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Informações</string>
     </property>
     <widget class="QLineEdit" name="lineEditPreAutorizado">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>NAO</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>31</y>
        <width>100</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Pre-Autorizado?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>59</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Anunciar sua chegada?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAnunciarChegada">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>SIM</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>910</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string> Autorizado Por</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>27</y>
       <width>50</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Morador</string>
     </property>
    </widget>
    <widget class="QRadLineEditCompleter" name="lineEditMorador">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>27</y>
       <width>360</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Digite o número do AP ou nome do morador</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>60</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>93</y>
       <width>32</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Torre</string>
     </property>
    </widget>
    <widget class="QRadLineEditEnter" name="lineEditAP">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>60</y>
       <width>120</width>
       <height>27</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadLineEditEnter" name="lineEditTorre">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>93</y>
       <width>120</width>
       <height>27</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadLineEditEnter" name="lineEditRamal">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>93</y>
       <width>120</width>
       <height>27</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>100</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ramal</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="topmenu">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>801</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 0, 133);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelStatus">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>541</width>
      <height>39</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Nova Visita</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PshBtnCancel">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>568</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Cancelar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PshBtnSave">
   <property name="geometry">
    <rect>
     <x>709</x>
     <y>568</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Salvar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRadLineEditCompleter</class>
   <extends>QLineEdit</extends>
   <header location="global">qradlineeditcompleter.h</header>
  </customwidget>
  <customwidget>
   <class>QRadLineEditEnter</class>
   <extends>QLineEdit</extends>
   <header location="global">qradlineeditenter.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditRG</tabstop>
  <tabstop>lineEditCPF</tabstop>
  <tabstop>lineEditVisitante</tabstop>
  <tabstop>lineEditMorador</tabstop>
  <tabstop>lineEditAP</tabstop>
  <tabstop>lineEditTorre</tabstop>
  <tabstop>lineEditRamal</tabstop>
  <tabstop>pushButtonBaterFoto</tabstop>
  <tabstop>PshBtnCancel</tabstop>
  <tabstop>PshBtnSave</tabstop>
  <tabstop>DtEdtData</tabstop>
  <tabstop>TmEdtHora</tabstop>
  <tabstop>DtEdtSaida</tabstop>
  <tabstop>TmEdtsaida_hora</tabstop>
  <tabstop>lineEditPreAutorizado</tabstop>
  <tabstop>lineEditAnunciarChegada</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
